pages/community/mission.md

```diff
@@ -34,3 +34,12 @@ to target our activities and actions to serving these three aspects:
   to support the establishment and expansion of RSE positions and
   groups within the research ecosystem including the material for
   making business justifications for RSEs.
+
+## 4. Diversity, Equality, and Inclusion
+
+  We believe Diversity, Equity, and Inclusion (DEI) are core values of the
+  US-RSE Association. A dedicated team of active members continue to lead the
+  development of a DEI program for our organization. DEI discussions are and
+  will continue to be an integral part of our community.
+
+  Please see our [full statement.]({{site.url}}/dei-mission)
```

Comment:
```suggestion
  Please see our [full statement.]({{site.url}}/dei-mission/)
```

Reply:
@vsoch Oh actually my comment above was after looking at this file. As earlier in this page we have the "inclusion definition" link without the `/`. So just to double check, I should go ahead and update that link as well to match the `/` suggestion here?

Reply:
Yeah generally the structure of our site has permalinks with a trailing slash: https://github.com/USRSE/usrse.github.io/blob/1569f845ce4ffbdcc873a49eae369ccffd67ca2e/_config.yml#L180